src/main/java/mods/railcraft/common/carts/EntityCartEnergyMFSU.java

```diff
@@ -0,0 +1,55 @@
+package mods.railcraft.common.carts;
+
+import ic2.api.item.IC2Items;
+import mods.railcraft.common.plugins.forge.LocalizationPlugin;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+
+public class EntityCartEnergyMFSU extends EntityCartEnergy {

```

Comment:
It should despawn the second the world get loaded... If Classic is installed the CESU Cart does not get registered in the EntityList. And if it is not installed the MFSU Cart does not get registered... You will get a lot of (If used) Skipping Entities but that should be it...